### ISSUE TITLE
Arbitrary values + template literal treated badly

### ISSUE THREAD

**Issue by juanlatorre** [francoismassart/eslint-plugin-tailwindcss#277]
**Describe the bug**
When using arbitrary values + template literals the plugin is breaking the classnames

**To Reproduce**
1. Use arbitrary value + template literal

**Expected behavior**

Leave arbitrary value as is without breaking the whole className.

**Screenshots**
![image](https://github.com/francoismassart/eslint-plugin-tailwindcss/assets/4494526/13df1535-8dcc-4859-bea4-a54cae0b3ab3)

**Environment (please complete the following information):**
 - OS: macOs
 - Softwares + version used:
     -  VSCode 1.82.1
     -  Node 18.15.0

**Additional context**

**eslint config file or live demo**
By providing a link to a live demo, a demo video or a github repo fixing the issue will be easier.

**Comment by andrewzey** [comment]
I'm running into this too

**Comment by vparpoil** [comment]
+1 on this

**Comment by alloalexandre** [comment]
Another workaround is to use `clsx` instead of template literal.

```tsx
import clsx from 'clsx';

const color = "red-500";
<div className={clsx("p-4", {
  [`bg-${color}`]: color
})} />
```

**Comment by francoismassart** [comment]
Hi, this will be fixed in v4 😇
